paasta_tools/contrib/paasta_update_soa_memcpu.py

```diff
@@ -231,25 +243,29 @@ def main(argv=None):
             m=serv['money'],
             x=serv['old_cpus'],
             y=serv['cpus'],
+            superregion_param=_get_dashboard_qs_param('superregion', serv['cluster'].replace('marathon-', '')),
```

Comment:
Can you call this 'cluster' throughout?

Reply:
it has to be superregion at least where it will be passed to sfx as a query string variable, but i could change my format string name